source/wp-content/themes/wporg-news-2021/block-template-parts/front-page/latest-posts.html

```diff
@@ -21,7 +21,7 @@ <h2 class="front__latest-posts-heading">Latest Posts</h2>
 	<!-- /wp:post-template -->
 
 	<!-- wp:query-pagination -->
-		<!-- wp:query-pagination-next {"label":"See All Posts"} /-->
+	<a href="./category/uncategorized/" class="wp-block-query-pagination-next">See All Posts</a>
```

Comment:
Open to suggestions for a quick hacky workaround we could use here for SotW - Uncategorized has the right effect but doesn't include all posts.